compiler/passes/scopeResolve.cpp

```diff
@@ -1922,8 +1922,8 @@ static void resolveEnumeratedTypes() {
         if (EnumType* type = toEnumType(first->var->type)) {
           if (SymExpr* second = toSymExpr(call->get(2))) {
             const char* name;
-            bool found = false;
-
+            //bool found = false;
+            /*
```

Comment:
I took out the code but left a comment. I don't think it will ever serve its purpose. I view the purpose of the commented out code as "throw an error if a bad enum symbol is used"

Let's say that I try to use .orange on a color enum that only has red, yellow, and blue. 
Maybe I've implemented enum type functions on color for green and purple that return blue + yellow or blue + red respectively. Is .orange an invalid color symbol or an invalid paren-less color method? It could be either. 

Catching .orange at scopeResolve would require being aware of whether it could be a valid function or not. If it isn't an enum symbol and gets passed along, when it reaches function resolution it will be caught there, and an (updated) error message will inform the user that they have an invalid enum symbol or call 'orange'.